README.md

```diff
@@ -34,10 +35,108 @@ Installation
 ember install ember-sinon-qunit
 ```
 
-
 Usage
 ------------------------------------------------------------------------------
 
+Ember-Sinon-QUnit supports two different versions of the Ember-QUnit API:
+
+1. The new QUnit hooks API, which takes a `hooks` object then wires up sandbox 
+creation and restoration to `beforeEach` and `afterEach` of the module, respectively.
+1. The classic API, which automatically wires up sandbox creation and restoration 
+to `QUnit.testStart` and `QUnit.testDone` respectively.
+
+### QUnit `hooks` API
```

Comment:
I'm torn as to whether we should even support this form. The cost of someone forgetting to add the `setupSinon` call is pretty high, in that they could leak stubs/mocks from another test into a subsequent one. 

Generally, the approach I took with `ember-sinon-sandbox` was to automatically wire things up so users fall into the pit of success. I'm open to the contrary.

Thoughts?

Reply:
Can you link to where you automatically wired things up? I didn't see anything like that documented for either addon, but I'm all for less work on the user.

Reply:
This is where we wire it up globally for `testStart`/`testDone` - https://github.com/scalvert/ember-sinon-sinoff/blob/master/addon-test-support/setup-global-sinon-sinoff.js